python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -1195,6 +1192,7 @@ def _reorder_maps(
         right_rows: int,
         rg: plc.Column,
         right_policy: plc.copying.OutOfBoundsPolicy,
+        maintain_order: Literal["none", "left", "right", "left_right", "right_left"],
```

Comment:
```suggestion
        maintain_order: Literal["left", "right", "left_right", "right_left"],
```

Or accept `"none"` but just return the input maps immediately.